tox.ini

```diff
@@ -108,6 +108,8 @@ deps=
     wheel
     build
     nox
+    click
+    rich
```

Comment:
total nit: but maybe we can just use directly our requirement file?

Reply:
good point, updated! i added requirements_tools to the base deps